lib/declarations.d.ts

```diff
@@ -230,6 +230,7 @@ interface IStaticConfig extends Config.IStaticConfig {
 	SOLUTION_SPACE_NAME: string;
 	DEBUG_PROJECT_FILE_NAME: string;
 	RELEASE_PROJECT_FILE_NAME: string;
+	triggerJsonSchemaValidation: boolean;
```

Comment:
Can we shorten the name a bit? I would suggest something similar to "validateProject" or "validateProjectSchema"